test/quantization/core/test_quantized_op.py

```diff
@@ -3661,6 +3661,93 @@ def test_embedding_bag_2d_indices(self):
         torch.testing.assert_close(result, qresult, atol=0.05, rtol=1e-3)
 
 
+class TestDynamicQuantizedConv(TestCase):
+    def _test_qconv_op_impl(self, q_mod, dq_op, dim, q_engine, dtype):
+        # The goal here is to show that the dynamic op is the same as
+        # calc params->quantize_input->quantized op->dequantize output
+        if q_engine not in torch.backends.quantized.supported_engines:
+            return
+        torch.backends.quantized.engine = q_engine
+        reduce_range = True
```

Comment:
This should be true only for fbgemm.

Reply:
I fixed this to work correctly and discard reduce_range in qnnpack